@here/harp-mapview/lib/geometry/TileGeometryCreator.ts

```diff
@@ -1667,4 +1673,133 @@ export class TileGeometryCreator {
+            tile.projection.projectPoint(
+                new GeoCoordinates(tile.geoBox.north, tile.geoBox.west),
+                northWestPoint
+            );
+            tile.projection.projectPoint(
+                new GeoCoordinates(tile.geoBox.south, tile.geoBox.east),
+                southEastPoint
+            );
+
+            // Set the plane distance to the difference between the plane's center and the tile's
+            // center in world space.
+            rightPlaneDistance.copy(eastPoint).sub(tile.center);
+            leftPlaneDistance.copy(westPoint).sub(tile.center);
+            topPlaneDistance.copy(northPoint).sub(tile.center);
+            bottomDistance.copy(southPoint).sub(tile.center);
+
+            // Compute the plane equation from the three relevant coplanar points for each plane (in
+            // counter-clockwise order), and extract their normals.
+            rightPlaneNormal.copy(
+                new THREE.Plane().setFromCoplanarPoints(southEastPoint, eastPoint, northEastPoint)
+                    .normal
+            );
+            leftPlaneNormal.copy(
+                new THREE.Plane().setFromCoplanarPoints(northWestPoint, westPoint, southWestPoint)
+                    .normal
+            );
+            topPlaneNormal.copy(
+                new THREE.Plane().setFromCoplanarPoints(northEastPoint, northPoint, northWestPoint)
+                    .normal
+            );
+            bottomNormal.copy(
+                new THREE.Plane().setFromCoplanarPoints(southWestPoint, southPoint, southEastPoint)
+                    .normal
+            );
+        }
+
+        // Initialize the planes.
+        const tileP0 = new THREE.Plane(rightPlaneNormal, 0);
+        const tileP1 = new THREE.Plane(leftPlaneNormal, 0);
+        const tileP2 = new THREE.Plane(topPlaneNormal, 0);
+        const tileP3 = new THREE.Plane(bottomNormal, 0);
+
+        // Setup the object onBeforeRender callback to update the clippingPlanes at runtime.
+        const tilePosition = new THREE.Vector3();
+        const tileRightPosition = new THREE.Vector3();
+        const tileLeftPosition = new THREE.Vector3();
+        const tileTopPosition = new THREE.Vector3();
+        const tileBottomPosition = new THREE.Vector3();
+        object.onBeforeRender = (_renderer, _scene, _camera, _geometry, _material) => {
+            if (_material.clippingPlanes === null) {
+                _material.clippingPlanes = [tileP0, tileP1, tileP2, tileP3];
+            }
+
+            // Transform the tile position to be relative to the camera (worldCenter).
+            tilePosition.copy(tile.center);
+            tilePosition.x += tile.projection.worldExtent(0, 0).max.x * tile.offset;
+            tilePosition.sub(mapView.worldCenter);
+
+            // Translate the clipping planes to the new tile camera-relative position.
+            _material.clippingPlanes[0] = tileP0
+                .clone()
```

Comment:
as above, you don't need to clone, you can use some prestored planes in the class.